packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { SynapseConstants } from '../../utils'
 import {
-  generateQueryFilterFromSearchParams,
+  getAdditionalFilters,
```

Comment:
Renamed generateQueryFilterFromSearchParams() to getAdditionalFilters() since it derives the additionalFilters field value based on what it finds in local storage as well as the given search parameters.